tools/web-fuzzing-introspection/app/webapp/helper/function_helper.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2024 Fuzz Introspector Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+from typing import Dict, List, Any
+
+from .. import models
+
+
+def process_functions(target_list: List[models.Function], project_name: str,
```

Comment:
Could we make this more precise in the name? A lot of the logic in the web app is about "processing functions"